### PR TITLE
Functions get and clean for continuous water temperature data. Also r…

### PR DESCRIPTION
**Pull request by catarfish**
…aw and cleaned data files, as well as station file.

### CODE REVIEW

**Review by ryanpeek — CHANGES_REQUESTED** [review]
This looks good. I'd recommend commenting out or removing the mapview lines in the `f_clean_watertemp.R` script. Generally the visualization stuff can be done outside of the function, because it can bog things down. Let me know if you want me to merge as is (and we can worry about it later), or if you want me to wait to merge until after you push up that small edit.

File: scripts/functions/f_clean_watertemp.R [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    # filter to 1998 to current
+    dplyr::filter(date >= as.Date("1998-01-01"))
+
+  # filter out certain latitudes and longitudes
+  stations <- readr::read_csv(stations_file)
+
+  stations_sf <- stations %>%
+    sf::st_as_sf(coords=c("Longitude","Latitude"), crs=4326, remove=FALSE)
+
+  mapview::mapView(stations_sf)
```
Comment (ryanpeek) [inline scripts/functions/f_clean_watertemp.R:28]: suggest removing this bit for the function.

File: scripts/functions/f_clean_watertemp.R [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+  stations_filtered <- stations %>%
+    dplyr::filter(Latitude <38.5 & Latitude > 38 &
+                    Longitude >-121.8 &
+                    !(County %in% c("San Joaquin", "Contra Costa"))&
+                    Basin != "San Joaquin R" &
+                    Station != "BKS") %>%
+    janitor::clean_names()
+
+  stations_filtered_sf <- stations_filtered %>%
```
Comment (ryanpeek) [inline scripts/functions/f_clean_watertemp.R:38]: same here, I'd remove anything that isn't used and returned in the `return` call...so `stations_filtered_sf` can get commented out or deleted too.

**Comment by catarfish** [comment]
Hi Ryan,

Commented them out and pushed, so I think you can go ahead and merge now.

**Pull request merged**